modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -327,6 +330,13 @@ protected function register_controls() {
 				'fit' => esc_html__( 'Fit to content', 'elementor' ),
 				'height' => esc_html__( 'Height', 'elementor' ),
 			],
+			'selectors_dictionary' => [
+				'fit' => '--tabs-v2-overflow-x: initial; --tabs-v2-overflow-y: initial;',
```

Comment:
If you have this as a default value, why do you need to set this as default in the CSS as well?

Reply:
css variable removed